enterprise-suite/scripts/install-es.sh

```diff
@@ -151,13 +161,49 @@ else
     should_upgrade=false
 fi
 
-#echo "CREDS pre-use: $( ls -l $CREDS )"
-if [ "true" == "$should_upgrade" ]; then
-    debug helm upgrade "$ES_HELM_NAME" "$chart_ref" \
-        "$HELM_CREDENTIALS_ARG" \
-        $@
+if [ "true" == "$should_export" ]; then
+    # We need to grab any --version setting and pull it out of the command line args
+    VERSION_ARG=""
+    REST=()
+    if [ "true" = "$has_version" ] ; then
+        while [[ $# -gt 0 ]]
+        do
+            arg=$1
+            case $arg in
+                # Seems helm will accept "--version=foo" as well as the documented "--version foo"
+                --version=*)
+                    VERSION_ARG="$arg"
+                    shift
+                    ;;
+                --version)
+                    VERSION_ARG="--version $2"
+                    shift
+                    shift
+                    ;;
+                *)
+                    REST+=("$arg")
+                    shift
+                    ;;
+            esac
+        done
+        set -- "${REST[@]}" # restore the other positional parameters
+    fi
+
+    TDIR=$( mktemp -d 2>/dev/null || mktemp -d -t 'install-es-tdir' )
+    debug helm fetch -d $TDIR "$VERSION_ARG" "$chart_ref"
+    debug helm template --name "$ES_HELM_NAME" --namespace "$ES_NAMESPACE" \
+        "$@" \
+        $TDIR/${ES_CHART}*.tgz > "$ES_EXPORT_FILE"
 else
-    debug helm install "$chart_ref" --name="$ES_HELM_NAME" --namespace="$ES_NAMESPACE" \
-        "$HELM_CREDENTIALS_ARG" \
-        $@
+    # Get credentials
+    import_credentials
```

Comment:
This means the resources won't have the secret they need. Is that intentional? Why not just include the secret for them to use?

Reply:
Very intentional after getting knuckles rapped by @dbrinegar when I _did_ include them.  Turns out that the credentials are easily decipherable with a few base64 decodes.

My plan is to add a flag or YAEV (yet another env var) to generate the appropriate `kubectl` command.  That way the user could do something like:
`eval $(install-es.sh -c)``
to get their credentials installed in the cluster without having to embed them in the yaml.  Thoughts?

Reply:
See commit https://github.com/lightbend/helm-charts/pull/150/commits/7c5bc0b3bef0c7319f7ee82b080cd1808e28e90b

The intent is that a user would do:
` eval $(ES_EXPORT_CREDS=true  ./scripts/install-es.sh)`
(After which they could apply the yaml from ES_EXPORT_YAML.)

Reply:
Sorry, I still don't understand the problem. What's wrong with printing out the secret resource for users? It's not being published anywhere. It seems really convenient to provide the entire resource set.

Reply:
Requiring users to use `kubectl` is not ideal - it requires a Kubernetes cluster to be up. It would be better if they had the yaml file they could apply or store in whatever way they desire. Idea here is for maximum flexibility for the user. If we're worried that users don't understand that a `Secret` contains secure info, then we can document that as a note.

Reply:
See general comments in this PR for response to your last two comments here.  In a nutshell:
- Can now export either 'console' yaml or 'creds' yaml.
- No more `kubectl` stuff.